cpp/test/CMakeLists.txt

```diff
@@ -44,12 +45,17 @@ add_executable(
   ${CMAKE_CURRENT_BINARY_DIR}/poisson.c
 )
 target_link_libraries(unittests PRIVATE Catch2::Catch2WithMain dolfinx)
+# UUID requires bcrypt to be linked on Windows, broken in vcpkg.
+# https://github.com/microsoft/vcpkg/issues/4481
+if (WIN32)
+  target_link_libraries(unittests PRIVATE bcrypt)
+endif()
 target_include_directories(
   unittests PRIVATE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
 )
-target_compile_options(
-  unittests PRIVATE $<$<COMPILE_LANGUAGE:CXX>:-Wall -Werror -Wextra -pedantic>
-)
+#target_compile_options(
```

Comment:
Uncomment or remove.

Reply:
My view is we should only play with Developer flags on Unix. All others should be left default.